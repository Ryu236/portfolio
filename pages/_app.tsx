import { AppProps } from 'next/app'
import { GoogleAnalytics, usePageView } from '../lib/gtag'
import '../styles/globals.css'

const App = ({ Component, pageProps }: AppProps) => {
  usePageView()

  return (
    <>
      <GoogleAnalytics />
      <Component {...pageProps} />
    </>
  )
}

export default App
