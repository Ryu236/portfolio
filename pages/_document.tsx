import Document, {
  DocumentContext,
  Head,
  Html,
  Main,
  NextScript,
} from 'next/document'

import { GA_ID } from '../lib/gtag'

export default class MyDocument extends Document {
  static async getInitialProps(ctx: DocumentContext) {
    const initialProps = await Document.getInitialProps(ctx)

    return initialProps
  }

  render() {
    return (
      <Html lang="en" prefix="og: https://ogp.me/ns/profile#">
        <Head>
          <link rel="icon" href="/favicon.ico" />
          <meta property="og:title" content="ryu236.com" />
          <meta property="og:type" content="profile" />
          <meta property="og:url" content="https://www.ryu236.com/" />
          <meta property="og:image" content="/icon.png" />
          <meta property="profile:username" content="Ryu236" />
          {/* Google Analytics */}
          <script
            async={true}
            src={`https://www.googletagmanager.com/gtag/js?id=${GA_ID}`}
          />
          <script
            dangerouslySetInnerHTML={{
              __html: `
                  window.dataLayer = window.dataLayer || [];
                  function gtag(){dataLayer.push(arguments);}
                  gtag('js', new Date());
                  gtag('config', '${GA_ID}', {
                    page_path: window.location.pathname,
                  });
                `,
            }}
          />
        </Head>
        <body>
          <Main />
          <NextScript />
        </body>
      </Html>
    )
  }
}
