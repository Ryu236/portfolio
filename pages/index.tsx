import React from 'react'
import Head from 'next/head'
import Image from 'next/image'

let isDarkMode: boolean
let theme: string

const SwitchTheme = function () {
  theme = document.documentElement.dataset.theme
  if (theme === 'dark') {
    isDarkMode = false
    document.documentElement.dataset.theme = 'light'
  } else {
    isDarkMode = true
    document.documentElement.dataset.theme = 'dark'
  }
}

const GithubIcon = function () {
  if (isDarkMode) {
    return <Image src="/github_icon_large_white.svg" width={60} height={60} />
  } else {
    return <Image src="/github_icon_large_dark.svg" width={60} height={60} />
  }
}

const BlogIcon = function () {
  if (isDarkMode) {
    return <Image src="/hatenablog-logo.svg" width={60} height={60} />
  } else {
    return <Image src="/hatenablog-logo.svg" width={60} height={60} />
  }
}

class Home extends React.Component {
  render() {
    return (
      <div className="container">
        <Head>
          <title>ryu236.com</title>
          <link rel="icon" href="/favicon.ico" />
        </Head>

        <main>
          <h1 className="title">Ryu236</h1>

          <p className="description">
            Hi, my name is Ryutaro Kobayashi.
            <br />I am a software engineer in Japan.
          </p>

          <div className="switch">
            <input id="toggle" type="checkbox" onChange={SwitchTheme} />
            <label htmlFor="toggle">Switch theme</label>
          </div>

          <div className="grid">
            <a href="https://blog.ryu236.com" className="card">
              <BlogIcon />
              <h3>Blog &rarr;</h3>
              <p>This is my blog. This blog is written by Japanese.</p>
            </a>

            <a href="https://github.com/Ryu236/" className="card">
              <GithubIcon />
              <h3>GitHub &rarr;</h3>
              <p>This is my github page. You can see my projects and code.</p>
            </a>

            <a href="https://twitter.com/Ryuk236" className="card">
              <Image
                src="/Twitter_Social_Icon_Circle_Color.svg"
                width={60}
                height={60}
              />
              <h3>Twitter &rarr;</h3>
              <p>My twitter id is Ryuk236. Please follow me!</p>
            </a>

            <a href="https://www.wantedly.com/id/ryu236" className="card">
              <Image src="/wantedly_mark.svg" width={60} height={60} />
              <h3>Wantedly &rarr;</h3>
              <p>My profile page in Wantedly. Please check this.</p>
            </a>
          </div>
        </main>

        <footer>
          <a
            href="https://twitter.com/Ryuk236"
            target="_blank"
            rel="noopener noreferrer"
          >
            Â© 2021 Ryutaro Kobayashi
          </a>
        </footer>

        <style jsx>{`
          .container {
            min-height: 100vh;
            flex-direction: column;
            justify-content: center;
            align-items: center;
          }

          main {
            padding: 5rem 0;
            flex: 1;
            display: flex;
            flex-direction: column;
            justify-content: center;
            align-items: center;
          }

          footer {
            width: 100%;
            height: 100px;
            border-top: 1px solid var(--border-color);
            display: flex;
            justify-content: center;
            align-items: center;
          }

          footer img {
            margin-left: 0.5rem;
          }

          footer a {
            display: flex;
            justify-content: center;
            align-items: center;
          }

          a {
            color: inherit;
            text-decoration: none;
          }

          .title a {
            color: #0070f3;
            text-decoration: none;
          }

          .title a:hover,
          .title a:focus,
          .title a:active {
            text-decoration: underline;
          }

          .title {
            margin: 0;
            line-height: 1.15;
            font-size: 4rem;
          }

          .title,
          .description {
            text-align: center;
          }

          .description {
            line-height: 1.5;
            font-size: 1.5rem;
          }

          code {
            background: #fafafa;
            border-radius: 5px;
            padding: 0.75rem;
            font-size: 1.1rem;
            font-family: Menlo, Monaco, Lucida Console, Liberation Mono,
              DejaVu Sans Mono, Bitstream Vera Sans Mono, Courier New, monospace;
          }

          .grid {
            display: flex;
            align-items: center;
            justify-content: center;
            flex-wrap: wrap;

            max-width: 800px;
            margin-top: 3rem;
          }

          .card {
            margin: 1rem;
            flex-basis: 45%;
            padding: 1.5rem;
            text-align: left;
            color: inherit;
            text-decoration: none;
            border: 1px solid var(--border-color);
            border-radius: 10px;
            transition: color 0.15s ease, border-color 0.15s ease;
          }

          .card:hover,
          .card:focus,
          .card:active {
            color: var(--active-color);
            border-color: var(--active-color);
          }

          .card h3 {
            margin: 0 0 1rem 0;
            font-size: 1.5rem;
          }

          .card p {
            margin: 0;
            font-size: 1.25rem;
            line-height: 1.5;
          }

          .logo {
            height: 1em;
          }

          @media (max-width: 600px) {
            .grid {
              width: 100%;
              flex-direction: column;
            }
          }
        `}</style>

        <style jsx global>{`
          :root {
            --active-color: #0070f3;
            --border-color: #eaeaea;
          }
          [data-theme='dark'] {
            --bg-color: #121212;
            --text-color: #f4f4f6;
            --active-color: #369bff;
            --border-color: #5d6d7e;
          }

          html,
          body {
            padding: 0;
            margin: 0;
            font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
              Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue,
              sans-serif;
          }
          main,
          container,
          footer {
            background: var(--bg-color);
            color: var(--text-color);
          }

          * {
            box-sizing: border-box;
          }
        `}</style>
      </div>
    )
  }
}

export default Home
