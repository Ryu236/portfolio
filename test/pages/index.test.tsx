import React from 'react'
import { render, screen } from '../testUtils'
import { Home } from '../../pages/index'

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Home />, {})
    expect(asFragment()).toMatchSnapshot()
  })

  it('should render the description', () => {
    render(<Home />)

    const description = screen.getByText(
      /Hi, my name is Ryutaro Kobayashi.I am a software engineer in Japan./i
    )

    expect(description).toBeInTheDocument()
  })

  /*
    it('clicking button triggers alert', () => {
      const { getByText } = render(<Home />, {})
      window.alert = jest.fn()
      fireEvent.click(getByText('Test Button'))
      expect(window.alert).toHaveBeenCalledWith('With typescript and Jest')
    })
  */
})
