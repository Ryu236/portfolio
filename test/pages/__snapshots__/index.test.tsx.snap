// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col mx-auto min-h-screen dark:text-white dark:bg-black"
  >
    <main
      class="flex flex-col py-16 items-center"
    >
      <button
        class="items-center p-2 pb-1 bg-black dark:bg-white rounded-full"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-sun w-5 h-5"
          color="#ea580c"
          data-icon="sun"
          data-prefix="far"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M375.7 19.7c-1.5-8-6.9-14.7-14.4-17.8s-16.1-2.2-22.8 2.4L256 61.1 173.5 4.2c-6.7-4.6-15.3-5.5-22.8-2.4s-12.9 9.8-14.4 17.8l-18.1 98.5L19.7 136.3c-8 1.5-14.7 6.9-17.8 14.4s-2.2 16.1 2.4 22.8L61.1 256 4.2 338.5c-4.6 6.7-5.5 15.3-2.4 22.8s9.8 13 17.8 14.4l98.5 18.1 18.1 98.5c1.5 8 6.9 14.7 14.4 17.8s16.1 2.2 22.8-2.4L256 450.9l82.5 56.9c6.7 4.6 15.3 5.5 22.8 2.4s12.9-9.8 14.4-17.8l18.1-98.5 98.5-18.1c8-1.5 14.7-6.9 17.8-14.4s2.2-16.1-2.4-22.8L450.9 256l56.9-82.5c4.6-6.7 5.5-15.3 2.4-22.8s-9.8-12.9-17.8-14.4l-98.5-18.1L375.7 19.7zM269.6 110l65.6-45.2 14.4 78.3c1.8 9.8 9.5 17.5 19.3 19.3l78.3 14.4L402 242.4c-5.7 8.2-5.7 19 0 27.2l45.2 65.6-78.3 14.4c-9.8 1.8-17.5 9.5-19.3 19.3l-14.4 78.3L269.6 402c-8.2-5.7-19-5.7-27.2 0l-65.6 45.2-14.4-78.3c-1.8-9.8-9.5-17.5-19.3-19.3L64.8 335.2 110 269.6c5.7-8.2 5.7-19 0-27.2L64.8 176.8l78.3-14.4c9.8-1.8 17.5-9.5 19.3-19.3l14.4-78.3L242.4 110c8.2 5.7 19 5.7 27.2 0zM256 368a112 112 0 1 0 0-224 112 112 0 1 0 0 224zM192 256a64 64 0 1 1 128 0 64 64 0 1 1 -128 0z"
            fill="currentColor"
          />
        </svg>
      </button>
      <h1
        class="leading-3 sm:leading-5 text-4xl sm:text-5xl mt-16 sm:mt-16"
      >
        Ryu236
      </h1>
      <p
        class="text-2xl mt-16"
      >
        Hi, my name is Ryutaro Kobayashi.
        <br />
        I am a software engineer in Japan.
      </p>
      <div
        class="flex flex-row items-center justify-center max-w-4xl mt-16"
      >
        <a
          class="sm:mx-6 mx-3"
          href="https://github.com/Ryu236/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            filter="invert(1)"
            height="60px"
            viewBox="0 0 120.775 117.794"
            width="60px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              data-name="Group 1"
              id="Group_1"
              transform="translate(-290.221 -58.431)"
            >
              <path
                d="M350.609,58.431a60.394,60.394,0,0,0-19.091,117.69c3.018.558,4.126-1.311,4.126-2.906,0-1.439-.056-6.2-.082-11.243-16.8,3.654-20.345-7.125-20.345-7.125-2.747-6.979-6.705-8.836-6.705-8.836-5.479-3.748.413-3.671.413-3.671,6.064.426,9.257,6.224,9.257,6.224,5.386,9.231,14.127,6.563,17.573,5.02.542-3.9,2.107-6.568,3.834-8.076-13.413-1.526-27.513-6.705-27.513-29.844A23.37,23.37,0,0,1,318.3,99.456c-.627-1.522-2.694-7.663.585-15.982,0,0,5.071-1.622,16.611,6.191a57.254,57.254,0,0,1,30.243,0c11.526-7.813,16.59-6.191,16.59-6.191,3.287,8.319,1.219,14.46.592,15.982a23.324,23.324,0,0,1,6.215,16.208c0,23.195-14.127,28.3-27.574,29.8,2.166,1.875,4.1,5.549,4.1,11.183,0,8.08-.07,14.583-.07,16.572,0,1.608,1.087,3.491,4.148,2.9A60.4,60.4,0,0,0,350.609,58.431Z"
                data-name="Path 2"
                fill="#0f0c0d"
                fill-rule="evenodd"
                id="Path_2"
              />
              <path
                d="M313.093,145.134c-.133.3-.6.391-1.035.185s-.684-.607-.542-.907.6-.4,1.04-.189.689.611.537.911Zm-.743-.55"
                data-name="Path 3"
                fill="#0f0c0d"
                id="Path_3"
              />
              <path
                d="M315.539,147.863c-.288.266-.851.142-1.233-.279a.923.923,0,0,1-.177-1.255c.3-.267.843-.142,1.239.279s.472.984.171,1.255Zm-.576-.618"
                data-name="Path 4"
                fill="#0f0c0d"
                id="Path_4"
              />
              <path
                d="M317.92,151.34c-.37.258-.975.017-1.349-.52s-.37-1.182.008-1.44.971-.025,1.35.507.369,1.191-.009,1.453Zm0,0"
                data-name="Path 5"
                fill="#0f0c0d"
                id="Path_5"
              />
              <path
                d="M321.182,154.7c-.331.365-1.036.266-1.552-.232a1.161,1.161,0,0,1-.343-1.543c.335-.365,1.044-.262,1.564.232s.684,1.181.331,1.543Zm0,0"
                data-name="Path 6"
                fill="#0f0c0d"
                id="Path_6"
              />
              <path
                d="M325.682,156.652c-.146.472-.825.687-1.509.486s-1.13-.761-.992-1.238.824-.7,1.513-.485,1.13.755.988,1.237Zm0,0"
                data-name="Path 7"
                fill="#0f0c0d"
                id="Path_7"
              />
              <path
                d="M330.624,157.013c.017.5-.563.911-1.281.92s-1.306-.387-1.314-.877.567-.911,1.289-.924,1.306.387,1.306.881Zm0,0"
                data-name="Path 8"
                fill="#0f0c0d"
                id="Path_8"
              />
              <path
                d="M335.223,156.231c.086.485-.413.984-1.126,1.117s-1.35-.172-1.439-.653c-.087-.5.421-1,1.121-1.126s1.353.168,1.444.662Zm0,0"
                data-name="Path 9"
                fill="#0f0c0d"
                id="Path_9"
              />
            </g>
          </svg>
        </a>
        <a
          class="sm:mx-6 mx-3"
          href="https://twitter.com/Ryuk236"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="60px"
            viewBox="25 -60 230 400"
            width="60px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g>
              <circle
                cx="140"
                cy="140"
                fill="#071C25FF"
                r="200"
              />
            </g>
            <g
              transform="scale(0.23,0.23)"
            >
              <path
                d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"
                fill="white"
              />
            </g>
          </svg>
        </a>
        <a
          class="sm:mx-6 mx-3"
          href="https://blog.ryu236.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="60px"
            viewBox="0 0 100 100"
            width="60px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M50.1 100.2C22.5 100.2 0 77.7 0 50.1S22.5 0 50.1 0s50.1 22.5 50.1 50.1c-.1 27.6-22.5 50.1-50.1 50.1zm0-93.8C26 6.4 6.4 26 6.4 50.1S26 93.8 50.1 93.8s43.7-19.6 43.7-43.7S74.2 6.4 50.1 6.4z"
              fill="#EEE"
            />
            <path
              d="M55.9 25.3c-2.4-5-3.9-9.8-4.7-12.6v35.5c1.2.5 2.1 1.6 2.1 3 0 1.8-1.5 3.3-3.3 3.3-1.8 0-3.3-1.5-3.3-3.3 0-1.5 1-2.7 2.3-3.1V12.7c-.8 2.8-2.2 7.6-4.7 12.6-3.8 7.8-8.6 14.6-8.6 14.6l3.1 41.8s2.9 3.2 11.3 3.2 11.3-3.2 11.3-3.2l3.1-41.8s-4.8-6.8-8.6-14.6z"
              fill="#EEE"
            />
          </svg>
        </a>
        <a
          class="sm:mx-3"
          href="https://www.wantedly.com/id/ryu236"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="85px"
            id="Logo"
            viewBox="0 0 500 394"
            width="85px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <defs />
            <circle
              class="cls-1"
              cx="375"
              cy="122.95"
              fill="#21bddb"
              r="38.98"
            />
            <path
              class="cls-2"
              d="M217.17,234.77c-2.34-1.52-9-10.45-27.42-54.44-1.15-2.76-2.22-5.14-3.23-7.18l-3.77-9.08L150.47,86.12H85.89l32.29,77.95,32.29,78,29.82,72a2.68,2.68,0,0,0,4.94,0l32.45-77.68A1.34,1.34,0,0,0,217.17,234.77Z"
              fill="#fff"
              fill-rule="evenodd"
            />
            <path
              class="cls-2"
              d="M338.15,234.77c-2.34-1.52-9-10.45-27.42-54.44-1.15-2.76-2.23-5.14-3.24-7.19l-3.75-9.07L271.45,86.12H206.87l32.29,77.95,32.29,78,29.82,72a2.68,2.68,0,0,0,4.94,0l32.45-77.67A1.36,1.36,0,0,0,338.15,234.77Z"
              fill="#fff"
              fill-rule="evenodd"
            />
          </svg>
        </a>
        <a
          class="sm:mx-6 mx-3"
          href="https://www.linkedin.com/in/ryu236/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="60px"
            viewBox="0 0 72 72"
            width="60px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="none"
              fill-rule="evenodd"
            >
              <path
                d="M8,72 L64,72 C68.418278,72 72,68.418278 72,64 L72,8 C72,3.581722 68.418278,-8.11624501e-16 64,0 L8,0 C3.581722,8.11624501e-16 -5.41083001e-16,3.581722 0,8 L0,64 C5.41083001e-16,68.418278 3.581722,72 8,72 Z"
                fill=""
              />
              <path
                d="M62,62 L51.315625,62 L51.315625,43.8021149 C51.315625,38.8127542 49.4197917,36.0245323 45.4707031,36.0245323 C41.1746094,36.0245323 38.9300781,38.9261103 38.9300781,43.8021149 L38.9300781,62 L28.6333333,62 L28.6333333,27.3333333 L38.9300781,27.3333333 L38.9300781,32.0029283 C38.9300781,32.0029283 42.0260417,26.2742151 49.3825521,26.2742151 C56.7356771,26.2742151 62,30.7644705 62,40.051212 L62,62 Z M16.349349,22.7940133 C12.8420573,22.7940133 10,19.9296567 10,16.3970067 C10,12.8643566 12.8420573,10 16.349349,10 C19.8566406,10 22.6970052,12.8643566 22.6970052,16.3970067 C22.6970052,19.9296567 19.8566406,22.7940133 16.349349,22.7940133 Z M11.0325521,62 L21.769401,62 L21.769401,27.3333333 L11.0325521,27.3333333 L11.0325521,62 Z"
                fill="#FFF"
              />
            </g>
          </svg>
        </a>
      </div>
      <article
        class="prose prose-h3:mt-0 lg:prose-p:mt-16 prose-p:mt-10 lg:prose-xl text-left mt-24 dark:text-white"
      >
        <h1
          class="text-center dark:text-white"
        >
          Resume
        </h1>
        <h2
          class="text-center dark:text-white"
        >
          Experience
        </h2>
        <div
          class="mx-4"
        >
          <div>
            <p>
              Mar 2023 - Present
            </p>
            <div
              class="ml-8"
            >
              <h3
                class="dark:text-white"
              >
                UPSIDER, Inc.
              </h3>
              Tech lead & Fullstack Engineer 
              <br />
              Development of the prevention fraudulent use transactions. 
              <br />
              Previously developed web applications, payment card processing systems.
            </div>
          </div>
          <div>
            <p>
              Apr 2023 - Jan 2024
            </p>
            <div
              class="ml-8"
            >
              <h3
                class="dark:text-white"
              >
                Shiftbase, Inc.
              </h3>
              Backend Engineer（side job） 
              <br />
              Development of the web application.
            </div>
          </div>
          <div>
            <p>
              Apr 2019 - Feb 2023
            </p>
            <div
              class="ml-8"
            >
              <h3
                class="dark:text-white"
              >
                DMM.com LLC
              </h3>
              Software Engineer 
              <br />
              Development, maintenance and operation of the payment platform.
            </div>
          </div>
          <div>
            <p>
              May 2021 - Jan 2022
            </p>
            <div
              class="ml-8"
            >
              <h3
                class="dark:text-white"
              >
                GANGAN, Inc.
              </h3>
              Backend Engineer（side job） 
              <br />
              Development of the application.
            </div>
          </div>
          <div>
            <p>
              Sep 2018 - Sep 2018 
            </p>
            <div
              class="ml-8"
            >
              <h3
                class="dark:text-white"
              >
                Eureka, Inc.
              </h3>
              Internship 
              <br />
              Development of Android application.
            </div>
          </div>
          <div>
            <p>
              Sep 2017 - Sep 2017 
            </p>
            <div
              class="ml-8"
            >
              <h3
                class="dark:text-white"
              >
                Sony Corporation
              </h3>
              Internship 
              <br />
              Development of embedded system.
            </div>
          </div>
        </div>
        <h2
          class="text-center mt-20 dark:text-white"
        >
          Education
        </h2>
        <div
          class="mx-4"
        >
          <div>
            <p>
              Apr 2017 - Mar 2019 
            </p>
            <div
              class="ml-8"
            >
              <h3
                class="dark:text-white"
              >
                Shinshu University
              </h3>
              Japan 
              <br />
              Master of Science and Technology 
              <br />
              Electrical and Computer Engineering Division
            </div>
          </div>
          <div>
            <p>
              Apr 2013 - Mar 2017 
            </p>
            <div
              class="ml-8"
            >
              <h3
                class="dark:text-white"
              >
                Shinshu University
              </h3>
              Japan 
              <br />
              Bachelor of Computer Science and Engineering
            </div>
          </div>
        </div>
      </article>
    </main>
    <footer
      class="w-full h-20 flex justify-center border-t mt-10"
    >
      <a
        class="flex items-center"
        href="https://twitter.com/Ryuk236"
        rel="noopener noreferrer"
        target="_blank"
      >
        © 2021 Ryu236
      </a>
    </footer>
  </div>
</DocumentFragment>
`;
